src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+public enum Operation {
```

Comment:
it should be the inner enum in FruitTransaction class. See task description 
![](https://user-images.githubusercontent.com/32339836/221908622-b5a0245d-1fc3-4727-b07e-7a3eb6eb5702.png)